Sources/NIOPosix/System.swift

```diff
@@ -111,7 +111,7 @@ private let sysIfNameToIndex: @convention(c) (UnsafePointer<CChar>?) -> CUnsigne
 private let sysSocketpair: @convention(c) (CInt, CInt, CInt, UnsafeMutablePointer<CInt>?) -> CInt = socketpair
 #endif
 
-#if canImport(Glibc)
+#if os(Linux) && !canImport(Musl)
```

Comment:
@MaxDesiatov, I don't know why you changed this for Musl, these functions aren't there? Because this change you made doesn't define them for Musl at all.

Reply:
@MaxDesiatov, do you define these for Musl? If so, feel free to add a commit to this pull with those Musl defintions, and we can get that fix in too.